internal/persistence/persistors/mysqlstore/store_organization.go

```diff
@@ -0,0 +1,290 @@
+package mysqlstore
+
+import (
+	"context"
+	"fmt"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/persistence"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/assets/mysqlmodel"
+	"github.com/dembygenesis/local.tools/internal/persistence/database_helpers/mysql/mysqltx"
+	"github.com/dembygenesis/local.tools/internal/sysconsts"
+	"github.com/friendsofgo/errors"
+	"github.com/volatiletech/sqlboiler/v4/boil"
+	"github.com/volatiletech/sqlboiler/v4/queries/qm"
+)
+
+func (m *Repository) DeleteOrganization(
+	ctx context.Context,
+	tx persistence.TransactionHandler,
+	id int,
+) error {
+	ctxExec, err := mysqltx.GetCtxExecutor(tx)
+	if err != nil {
+		return fmt.Errorf("get ctx exec: %v", err)
+	}
+
+	entry := &mysqlmodel.Organization{
+		ID:       id,
+		IsActive: false,
+	}
+	if _, err = entry.Update(ctx, ctxExec, boil.Whitelist("is_active")); err != nil {
```

Comment:
Use the mysqlmodel consts to get the "is_active" name, [example](https://github.com/dembygenesis/theoverwatchtools.cli/blob/33dd61bccf6819341626cb14f1736f6bf80589f2/internal/persistence/persistors/mysqlstore/store_category.go#L60).

Reply:
This is now resolved.